pkg/sql/sem/tree/testdata/eval/builtins

```diff
@@ -82,3 +82,23 @@ eval
 ceiling(-1.5::float)
 ----
 -1.0
+
+eval
+length('1011'::bit(4))
+----
+4
+
+eval
+bit_length('1011'::bit(4))
+----
+4
+
+eval
+octet_length('1011'::bit(4))
```

Comment:
sorry i'm confused why this returns something different than below

Reply:
this one is 4 bits, which needs 1 byte.
the below is 12 bits, which needs 2 bytes.

Reply:
but its the same bit string idgi

Reply:
a bit string is represented as bytes in memory

Reply:
oh i was confused by the size parameter in `BIT`